lib/geared_pagination/portions/portion_at_cursor.rb

```diff
@@ -0,0 +1,128 @@
+require 'geared_pagination/ratios'
+require 'geared_pagination/cursor'
+require 'active_support/core_ext/array'
+
+module GearedPagination
+  class PortionAtCursor
+    attr_reader :cursor, :orders, :ratios
+    delegate :page_number, to: :cursor
+
+    def initialize(cursor: Cursor.new, ordered_by:, per_page: Ratios.new)
+      @cursor, @orders, @ratios = cursor, ordered_by, per_page
+    end
+
+    def from(scope)
+      selection_from(scope).order(orderings).limit(limit)
```

Comment:
Might want to use `.reorder` to make sure the cursor fully controls the ordering. Right now, it aggregates existing order clauses in the base relation, which might cause trouble since those are likely to be oversights (e.g: an ordered association).

If we do this, we could show a warning about "order being ignored" when `scope.order_values.present?`.

Similarly, existing `limit` causes will be overridden and ignored (which is fine and expected), but maybe both a warning too (`scope.limit_value.present?`).

Reply:
Did it in 18a7c54 by raising an `ArgumentError`.